### DISCUSSION TITLE
Welcome to WebWowViewerCpp Discussions!

### DISCUSSION THREAD

**Discussion by Deamon87** [Deamon87/WebWowViewerCpp#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by drnoonien** [comment]
Hey, apologies if this is the wrong place for this but I have a question. I used this program to explore the Amirdrassil raid instance before the release of 10.2.

Is it possible to do the same now for Nerub-ar Palace that's available on the 11.0 beta?

I can open the application and select the beta build just fine, but there doesn't seem to be any 11.0 zones in the map selector, and the DB update fails on importing some `AreaTable`.

![](https://i.imgur.com/7qbvX7a.png)

**Reply by Deamon87** [reply]
Even though it's been 6 months, but new 0.3.0 release should be able to fix this issue

**Reply by drnoonien** [reply]
Works great, thanks!

**Comment by PetrSebik** [comment]
Hi @Deamon87, I love this viewer. But I am getting similar error to the comment above.
![image](https://github.com/user-attachments/assets/e29d96ac-9e14-49cc-ae93-cacfd8afdeac)

here is the console traceback
```
Opened local Casc Storage at C:\Program Files (x86)\World of Warcraft\_retail_\
Could not read fileDataId 1353545 from local
Could read fileDataId 1353545
```

I tried `_retail_` and `_classic_` in the version 0.31 and it gave me the same error.
In the older version 0.30 the `_retail_` works, but the classic still no luck.
